docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md

```diff
@@ -40,6 +40,25 @@ If Docker is not installed on the host computer, you need to install a recent ve
 
 If access to the Docker repositories for your Linux distribution is not available on the host computer, you may have to manually transfer the necessary RPM or DEB packages whose locations are specified in [Installing Docker in Airgapped Environments](https://community.replicated.com/t/installing-docker-in-airgapped-environments/81).
 
+{{< note title="Docker on Amazon Linux" >}}
```

Comment:
not sure it needs to be in a note...

Also, Amazon linux isn't listed as a supported OS on the prerequisites page https://deploy-preview-16629--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/#supported-linux-distributions (also, the replicated link in this section I linked is 404)